test-cases/gutenberg/group.md

```diff
@@ -0,0 +1,355 @@
+
+##### TC003
+
+### AppenderButton is render properly
+
+-   Add a `Group` block 
+-   Check if `AppenderButton` renders properly
+-   Check if `AppenderButton` is align with the content
+
+Expected look:
+ <<<SCREENSHOT>>
+
+--------------------------------------------------------------------------------
+
+##### TC004
+
+### Group AppenderButton allows to open Inserter and add new block to group
+
+-   Add a `Group` block 
+-   Check if you are able to open `Inserter` menu by press on `AppenderButton` inside `Group`
+-   Check if the block is added inside the `Group` before the `AppenderButton`
+-   Select a `Group` block once again
+-   Check if you are able to add another block
+-   Check if the block is added inside the `Group` before the `AppenderButton`
+
+--------------------------------------------------------------------------------
+
+##### TC005
+
+### DefaultAppender allows to open Inserter and add block outside the group
+
+-   Add a `Group` block 
+-   Having `Group` block selected press on `DefaultAppender`
+-   Check if the block is added outside after the `Group`
+
+<br />
+
+-   Having `Group` block selected long press on `DefaultAppender`
+-   Select where you want to add block after or before
+-   Check if the block is added outside after/before the `Group`
+
+--------------------------------------------------------------------------------
+
+##### TC006
+
+### DefaultAppender allows to open Inserter and add block inside the group
+
+-   Add a `Group` block 
+-   Add block inside the `Group`
+-   Having newly added block selected press on `DefaultAppender`
+-   Check if the block is added inside the `Group` after the selected block
+
+<br />
+
+-   Having newly added block selected long press on `DefaultAppender`
+-   Select where you want to add block after or before
+-   Check if the block is added inside the `Group` after/before the selected block
+
+--------------------------------------------------------------------------------
+
+##### TC007
+
+### Deep nesting is possible
+
+-   Add a `Group` block 
+-   Nest blocks inside multiple `Group` blocks (at least 3 levels deep)
+-   Check if app do not crashes
+
+--------------------------------------------------------------------------------
+
+##### TC008
+
+### Check if Group placeholder is render on RootList
+
+-   Add a `Group` block 
+-   Deselect
+-   Check if you are able to see `Group` placeholder
+
+Expected look:
+ << SCREENSHOT >>
+
+--------------------------------------------------------------------------------
+
+##### TC009
+
+### Check if Group placeholder is render in nested structure
+
+-   Add a `Group` block 
+-   Nest some `Group` inside
+-   Select top-most `Group` block in hierarchy
+-   Check if you are able to see `Group` placeholder of nested blocks wrapped with dashed border
+-   Go down in the hierarchy observing if margins is keept (the whole UI should remain the same)
```

Comment:
ok, this one is difficult to explain because if you add multiple empty groups(at least 4levels deep) and start to navigate-dow you will not see any changes when selecting 3 and 4 level. The UI look is exactly the same it that case

Reply:
I see the images attached but we need to put some marks on our visuals to show which distance we want to emphasize otherwise someone who is not familiar with this feature won't understand what you mean.

Reply:
I think after we extract margins to separate TC we can limit this one to check if placeholder is rendered. 

Having proper margins in TC007 we should also cover "margins is kept" case from this TC